source/partials/next_release/_release.md.erb

```diff
@@ -61,16 +99,20 @@
   "Mahesh Panchaksharaiah",
   "Marques Lee",
   "Naveen Bhaskar",
+  "Pranav Bansod",
+  "Rahul Pargaonkar",
   "Rajiesh N",
   "Senthil R",
-  "Tomasz Sętkowski",
-  "Varsha Varadarajan"
+  "Steven Streisguth",
+  "Varsha Varadarajan",
+  "Vishal Devgire",
```

Comment:
We are not adding helm contributors to release note as of now.

Reply:
Why not?

Reply:
So? We are naming people who have helped make GoCD better. A contribution to the GoCD helm chart irrespective of the helm chart release is something to be acknowledged.

Reply:
Sure, we will consider this for upcoming releases.

Reply:
@varshavaradarajan @bdpiparva  Running this:

```shell
git log --since="July 23, 2018" --pretty="tformat:%an" stable/gocd/ | sort | uniq
```

gives me a list of names of people who have contributed to the helm charts over the last release. If either of you wants to update the release notes with that, I can. Let me know.

Reply:
@arvindsv - please update the release note if possible or I will do it tomorrow morning. 

> We are naming people who have helped make GoCD better

What @varshavaradarajan is saying makes sense to me.

Reply:
@bdpiparva Done: https://github.com/gocd/www.go.cd/commit/c7c767db3b8f9c3e8120cdffabd13c1303c8f18e

Reply:
Haven't added a message about helm. Feel free to do that if you want, @varshavaradarajan.